documentation/API_documentation/API_definition.md

```diff
@@ -0,0 +1,192 @@
+- The Mobile Operator’s ID GW validates the request (i.e. that the SP has been registered with the Mobile Operator for the Mobile Connect Verified MSISDN service requested)
+- The ID GW optionally can check that the source is from a known IP range (white listed)
+- The Mobile Operator ID GW checks whether the User is registered for Mobile Connect
+- If the MSISDN is not yet registered for Mobile Connect, based on ID GW policies a new Mobile Connect account may be created “on-the-fly” for that MSISDN. 
+- The Mobile Operator retrieves the MSISDN from the IP Header (or uses an alternate mechanism to obtain the network-authenticated MSISDN for that device)
+
+In order to be able to share (or match) User information with a SP, the User must give their consent. 
+
+> **Figure 1** illustrates the option where the Mobile Operator ID GW captures User consent which, within the context of this use case, might be at initial registration for the SP service.
+>
+> User consent can also be captured by the SP, subject to the Mobile Operator’s ID GW consent policies and the contractual agreement with the SP. 
+>
+> Consent can also be long-lived where it exists for an extended period of time to provide a smoother User experience.
+>
+> If consent is not required (i.e. the User has already given their consent) then this step can be omitted.
+
+- An Authorization Code is then returned to the SP as part of a standard Device-Initiated flow and the SP can then retrieve the ID Token and Access Token by make a Token Request to the ID GW Token endpoint.
+- For Verified MSISDN Share the SP can then make a Resource Request to the Resource endpoint with the Access Token to fetch the MSISDN of the device. 
+- For Verified MSISDN Match the SP can fetch the MSISDN match result by making a Resource Request with the Access Token and including the appropriate attribute identifier and value within the mc_claims parameter:
+  - The attribute identifier is  “device_msisdn” with a value of the MSISDN in plain text if scope= ”mc_vm_match” was specified in the service request.
+  - The attribute identifier is “device_msisdn_hash”  with a value of the hash of the MSISDN if scope=”mc_vm_match_hash” was specified in the service request.
+- The Resource Server matches the attribute value with the previously extracted MSISDN or hash of the MSISDN extracted by the Authorization Server. If they match, the service returns a Boolean value “true” to the SP. 
+- If the verification fails, then it returns a Boolean value “false” to the SP. 
+
+
+### User Consent Management
+
+The Mobile Connect Verified MSISDN service is designed to be used as a fraud check so typically, User consent will be captured by the SP (at registration and /or included within standard terms and conditions) and the service will be processed in the background without any explicit User consent.
+
+## Verified MSISDN Service Specification
+
+### OIDC Authorization Request Parameters - scope
```

Comment:
Scope missing in the swagger. See previous comment

Reply:
this was fixed with last commit